docs/REFACTORING_PHASE5_PLAN.md

```
@@ -0,0 +1,713 @@
+|---------|---------|
+| `RecordForm.tsx` | ローカル定義 → インポート |
+| `RecordLogForm.tsx` | ローカル定義 → インポート |
+| `PracticeLogForm.tsx` | ローカル定義 → インポート |
+| `TeamTimeInputModal.tsx` | ローカル定義 → インポート |
+| `TeamEntrySection.tsx` | ローカル定義 → インポート |
+| `utils/formatters.ts` | 重複関数を削除、re-export |
+| `LapTimeDisplay.tsx` | ローカル定義 → インポート |
+
+### 推定工数
+
+**2-3時間**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)